services/src/main/scala/cromwell/pipeline/service/ProjectFileService.scala

```diff
@@ -0,0 +1,17 @@
+package cromwell.pipeline.service
+
+import cromwell.pipeline.datastorage.dto.FileContent
+import cromwell.pipeline.womtool.WomTool
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class ProjectFileService(womTool: WomTool)(implicit executionContext: ExecutionContext) {
+
+  def validateFile(fileContent: FileContent): Future[Either[ValidateError, String]] =
+    Future(womTool.validate(fileContent.content)).map {
+      case Left(value) => Left(ValidateError(value.head)) // thinking about change Left on Future.failed
+      case Right(_)    => Right("File is valid")
+    }
+}
+
+case class ValidateError(message: String) extends Exception(message)
```

Comment:
I think we should stop creating error classes in the service classes.
@ilyaonishenko do we have some `model` module, that contains all case classes that are used by all other modules?

Reply:
'model' module is WIP right now. It should be done in Ostap's MR.
Right now we have `dto` directory in repositories.

Reply:
OK, will move case class into it